### PR TITLE
Fix Automated Snyk PRs 🥽 

### PR DESCRIPTION
**Pull request by nickumia-reisys**
Related to
- https://github.com/GSA/data.gov/issues/3416
- #675 

References:
- https://docs.snyk.io/snyk-cli/configure-the-snyk-cli/environment-variables-for-snyk-cli
- https://docs.snyk.io/snyk-cli/commands/auth
- https://github.com/actions/runner-images
- https://yamllint.readthedocs.io/en/stable/disable_with_comments.html#disabling-checks-for-all-or-part-of-the-file
- https://docs.github.com/en/actions/learn-github-actions/environment-variables
- https://stackoverflow.com/a/12505057
- https://github.com/GSA/datagov-brokerpak-eks/blob/main/Makefile#L18

### CODE REVIEW

**Review by jbrown-xentity — APPROVED** [review]
I worry in general that this custom solution (while well thought out and detailed) will eventually break due to things outside our control (like pip changes), but for now this is great!

**Comment by nickumia-reisys** [comment]
> I worry in general that this custom solution (while well thought out and detailed) will eventually break due to things outside our control (like pip changes), but for now this is great!

That's the worry we carry as developers in the ever-changing world of technology.  But yes, we carry some burden of maintaining this.  I see this as part of our deployment pipeline which would always be a custom solution in any case.

File: tools/snyk-update.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    package, old_version = k.split('@')
+    new_version = v['upgradeTo'].split('@')[1]
+    print(package, old_version, new_version)
+
+    # Remove old version
+    os.system('sed -i "/%s/d" ckan/requirements.in' % (package + "==" + old_version))
+    os.system('sed -i "/%s/d" ckan/requirements.in' % (package + ">=" + old_version))
+
+    # Add new version
+    os.system("echo '%s' >> ckan/requirements.in" % (package + ">=" + new_version))
```
Comment (btylerburton) [inline tools/snyk-update.py:22]: do we want snyk PR's to be fuzzy versioned or fixed?

Comment (nickumia-reisys) [reply tools/snyk-update.py:22]: The "loose" `requirements.in` will be fuzzy, but the "definitive" `requirements.txt` will be fixed.

**Pull request merged**